introduction/index.rst

```diff
@@ -4,192 +4,155 @@ Introduction to the SOF Project
 
-What license does the firmware and SDK use?
-  The firmware is released using a standard BSD license with some parts
-  MIT. The SDK is GPL.
+#. Abstraction of host side DSP hardware to common API operations.
 
-Do I need to open source my firmware code changes?
-  No. The firmware BSD and MIT licensed code means you can keep code
-  changes private. Patches are always welcomed if do decide to open source
-  work.
+The Linux SOF ALSA/ASoC driver is upstream in Linux v5.2 onwards.
+
+
+Firmware Toolchain
+------------------
+
+GNU GCC can be used as a free SOF compiler alongside proprietary DSP vendor
+compilers. The choice of compiler is up to the user depending on features
+and budget. GCC complier is free.
+ 
+ 
+DSP Emulator
+------------
+
+Qemu can be used to provide a functional emulator to simultaneously trace and
+debug driver and DSP firmware code. Proprietary emulators are also available.
 
-What DSP architectures are supported?
-  |SOF| currently supports the Cadence/Tensilica Xtensa audio DSP
-  architecture and ISA.
+Emulation is also used within SOF CI for feature validation prior to merging
+new code.
 
-What host platforms are supported?
-  |SOF| currently supports the Intel Baytrail and Cherrytrail based
-  platforms. This includes devices like the MinnowBoard MAX and the ASUS
-  T100 laptop, but should also include any Baytrail or Cherrytrail based
-  devices that have the audio DSP enabled in the BIOS.
 
-  The code has also been designed to easily port to other host platform
-  architectures like ARM, MIPS etc.
+General FAQ
+===========
 
-How can I get involved?
-  Please join the developer mailing where new development features and
-  patches are discussed:
-  http://alsa-project.org/mailman/listinfo/sound-open-firmware
+What license does the firmware use ?
+  The firmware is released using a standard BSD 3 clause license with some
+  files released under MIT.
 
-What is the development model?
-  |SOF| has a similar development model to the Linux kernel. Patches are
-  discussed and posted on the mailing list before being merged. The
+Do I need to open source my firmware code changes ?
+  No. The firmware BSD and MIT licensed code means you can keep code
+  changes private. Patches are always welcomed if do decide to open source
+  work.
+
+What license does the host driver use ?  
+  Most of the host driver code is dual licensed BSD or GLPLv2 only 
+  (users choice). The part of the driver that is GPLv2 only is the Linux
+  integration layer at the top of the driver stack
+
+Do I need to open source my driver code changes ?
+  No for the bottom two layers of the driver stack. i.e. if you are porting the 
+  driver to another OS, these changes can be kept private. Please note that the
+  driver GPL source files are all Linux specific and should not be ported to
+  another OS.
+
+How can I get involved ?
+  There are several ways to get involved, only the github method is mandatory :-
+#. Join github and fork the project.
```

Comment:
Line 130-132: The numbered list needs to follow a carriage return and be indented to match the rest of the definition list content.